### DISCUSSION TITLE
Requirement: Pixel device with Google Tensor Chipset

### DISCUSSION THREAD

**Discussion by tchwpkgorg** [kyujin-cho/pixel-volte-patch#9] [Q&A] [answered]
Why does it need a Google Tensor Chipset?

Wondering if it could be used on older Pixel models.

**Comment by kyujin-cho** [comment, marked as answer]
Simple: I could only test it working on my Pixel 7 Pro. For compatibility on older Pixel phones please check https://github.com/kyujin-cho/pixel-volte-patch/discussions/5.
